apps/Launcher/qml/DemoLauncher/RenderingResourcesManager.js

```diff
@@ -49,9 +59,13 @@ Communicator.prototype.isRunning = function() {
     return this.currentStatus === SESSION_STATUS_RUNNING;
 }
 
-Communicator.prototype.sendRequest = function(method, module, command, body, callback) {
+Communicator.prototype.sendRequest = function(method, module, command, body, callback, omitSessionId) {
     var fullUrl = this.serviceUrl + '/' + module + '/' + command;
 
+    if (!omitSessionId) {
+        fullUrl += ('?session_id=' + this.sessionId);
+    }
```

Comment:
Generally when something is only used once throughout the whole codebase there's really no point to make a var for it.